qemu/tests/cfg/blockdev_commit_speed_limit.cfg

```diff
@@ -0,0 +1,36 @@
+- blockdev_commit_speed_limit:
+    type = blockdev_commit_speed_limit
+    virt_test_type = qemu
+    only Linux
+    device_tag = 'data'
+    images += " ${device_tag}"
+    force_create_image_data = yes
+    remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags_data = sn1
+
+    image_size_sn1 = 500M
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    rebase_mode = unsafe
+    qemu_force_use_drive_expression = no
+    commit_speed = 10
+    variants:
+        - negative_int:
+            invalid_commit_speed = -1
+            error_msg = "Parameter 'speed' expects a non-negative value"
+        - small_int:
+            invalid_commit_speed = 0
+        - large_int:
+            invalid_commit_speed = 100000
```

Comment:
If we change to use 1 for the 'small_int' test, we can increase the value 100000 to a more larger one, so the test can finish quicker.

Reply:
Okay,thanks.